examples/with-shell-commands/README.md

```diff
@@ -0,0 +1,44 @@
+# Turborepo starter with shell commands
+
+This is an official starter Turborepo meant for debugging, learning, and exploring.
+
+## Using this example
+
+Run the following command:
+
+```sh
+npx create-turbo@latest -e with-shell-commands
+```
+
+### For bug reproductions
```

Comment:
This should be linked to or added to the Turborepo bug template!

Reply:
Didn't know how much we loved our bug issue template but there's a couple additions (this being one of them) that I'd like to propose. 😄 

Just wanted to make sure the rest of the group thinks it's a good idea first.